draft-ietf-cbor-7049bis.md

```diff
@@ -33,12 +33,12 @@ author:
   org: ICANN
   email: paul.hoffman@icann.org
 normative:
-  ECMA262:
-    target: http://www.ecma-international.org/publications/files/ecma-st/ECMA-262.pdf
-    title: ECMAScript Language Specification 5.1 Edition
+  ECMA-252-2017:
```

Comment:
s/252/262/.

Reply:
Thanks good catch.    Perhaps it might be better to use a more common designation identifier like 'ECMAScript-2017'.